devops/debian/postinst

```diff
@@ -36,6 +36,11 @@ esac
 
 # Automatically added by dh_installsystemd/12.10ubuntu1
 if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
+	# MSFT change: binplace osconfig.toml for the azure-identity-service only if not present
```

Comment:
>MSFT change: [](http://example.com/codeflow?start=3&length=12)

Why would we comment with "MSFT change"? Please remove this and do not use anymore. There is no such convention.